README.md

```
@@ -0,0 +1,94 @@
+- 결제
+    - KakaoPay 연동 통한 결제 요청/승인
+    - 결제 실패 시 티켓 자동 취소
+
+- 알림
+    - 예매 성공/실패 시 Email or Discord 알림 전송
+    - Kafka 이벤트 기반 비동기 전송
+
+- 모니터링
+    - Prometheus + Grafana로 트래픽, 지표 시각화
+    - Loki + Promtail로 중앙 집중형 로그 수집
+ 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)